docs/api/cpp/quickstart.rst

```diff
@@ -0,0 +1,188 @@
+Quickstart Guide
+================
+
+First, create a cvc5 :cpp:class:`Solver <cvc5::api::Solver>` instance:
+
+.. literalinclude:: ../../../examples/api/cpp/quickstart.cpp
+     :language: cpp
+     :lines: 27
+
+We will ask the solver to produce models and unsat cores in the following,
+and for this we have to enable the following options.
+
+.. literalinclude:: ../../../examples/api/cpp/quickstart.cpp
+     :language: cpp
+     :lines: 31-32
+
+Next we set the logic.
+The simplest way to set a logic for the solver is to choose "ALL".
+This enables all logics in the solver.
+Alternatively, ``"QF_ALL"`` enables all logics without quantifiers.
+To optimize the solver's behavior for a more specific logic,
+use the logic name, e.g. ``"QF_BV"`` or ``"QF_AUFBV"``.
+
+.. literalinclude:: ../../../examples/api/cpp/quickstart.cpp
+     :language: cpp
+     :lines: 41
+
+In the following, we will define constraints of reals and integers.
+For this, we first query the solver for the corresponding sorts.
```

Comment:
Is `query` the right word? sounds a bit like a check-sat query or something.

Reply:
It is the right word, yes. You ask the solver for the sort.